### PR TITLE
NE css style guide

### PR DESCRIPTION
**Pull request by imfelix**
#### Please review.

**Selector Naming** is still Work In Progress. 

Guide can be viewed [here](https://github.com/net-engine/css-style-guide/tree/NE-css-style-guide)

### CODE REVIEW

**Comment by imfelix** [comment]
@Arkinsaal , @beauhankins , @lennonzf , @johnpmorris , @jverban , @jamesrplee , @Kayelm , @nmqanh , @Dissimulate - hey guys, was hoping if you guys could review this styleguide PR for me. :grimacing: 

The Selector Naming is still in progress, as I am still trying to figure out whether or not we should go full-on f(css). :confused:

File: README.md [hunk 1]
```diff
@@ -1,54 +1,83 @@
...
+The purpose of this _(or any)_ styleguide is to,
 
+ - set the standard for code quality across a codebase;
+ - promote consistency across codebases;
+ - give developers a feeling of familiarity across codebases;
+ - increase productivity.
+
+
+
+
```
Comment (ecoologic) [inline README.md]: talking about consistency... why all these blank lines?

File: README.md [hunk 2]
```diff
@@ -1,54 +1,83 @@
...
+### General Rules
+
+Note on the language:
+
+* **Don't** means don't do it unless you have good reason
+* **Avoid** means there should be a better option, do that.
+* **Use** means do it.
+
+
```
Comment (ecoologic) [inline README.md]: I blank line should be enough if you use the correct header level

**Comment by elliotthilaire** [comment]
This is fantastic. I'd like to suggest taking it to the next level and adding an accompanying linter configuration. One that follows your rules defined here, nothing more and nothing less.

This will save me embarrassment and you the effort when I try to css.

File: README.md [hunk 3]
```diff
@@ -59,37 +88,89 @@ Finally, properties are what give the selected elements of a rule declaration th
...
+* **Use** new line for closing braces `}` of rule declarations.
+* **Use** empty spaces before and after child selector
+  * E.g. `div > span` instead of `div>span`
+* **Don't** write multiple properties on one line
+* **Don't** write multiple selectors on one line
+* **Don't** use ID selectors
+
+#### Formatting
+
+* **Use** hyphen separated aka “spinal case” over camelCasing in selector class names
```
Comment (beauhankins) [inline README.md]: "... `spinal-case` over `camelCase` or `snake_case` ..."

**Comment by beauhankins** [comment]
Looks good Felix! :clap:

**Comment by ecoologic** [comment]
:+1:

**Pull request closed without merging**